modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -407,17 +409,19 @@ def decrypt_user_hash(rid, hboot_key, enc_hash, pass, default)
     des_k1, des_k2 = rid_to_key(rid)
 
     d1 = OpenSSL::Cipher.new('des-ecb')
+    d1.decrypt
     d1.padding = 0
     d1.key = des_k1
 
     d2 = OpenSSL::Cipher.new('des-ecb')
+    d2.decrypt
     d2.padding = 0
     d2.key = des_k2
 
-    d1o = d1.decrypt.update(okey[0, 8])
+    d1o = d1.update(okey[0, 8])
     d1o << d1.final
 
-    d2o = d2.decrypt.update(okey[8, 8])
+    d2o = d2.update(okey[8, 8])
```

Comment:
Variations of this code snippet seemed to exist in a few places, maybe we're missing a shared module?